app/models/compute_resources/foreman/model/libvirt.rb

```diff
@@ -230,6 +230,11 @@ def normalize_vm_attrs(vm_attrs)
 
     protected
 
+    def libvirt_connection_error
+      msg = N_('Unable to connect to libvirt due to: %s. Please make sure your libvirt compute resource is reachable and that you have appropriate access permissions.')
+      raise Foreman::Exception.new(msg, errors.full_messages.join(', '))
```

Comment:
I'm not sure how this passed PR tests, but it is now failing on jenkins - https://ci.theforeman.org/job/test_develop/3603/database=postgresql,ruby=2.3,slave=fast/testReport/junit/(root)/HostJSTest__create%20new%20host%20page/test_0003_choosing_a_hostgroup_with_compute_resource_works/